keyboards/m0lly/info.json

```
@@ -0,0 +1,97 @@
+{
+    "keyboard_name": "TKC M0LLY", 
+    "url": "", 
+    "maintainer": "qmk", 
+    "bootloader": "", 
+    "width": 19.5, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+				{"label":"~", "x":0, "y":0},
+				{"label":"!", "x":1, "y":0},
+				{"label":"@", "x":2, "y":0},
+				{"label":"#", "x":3, "y":0},
+				{"label":"$", "x":4, "y":0},
+				{"label":"%", "x":5, "y":0},
+				{"label":"^", "x":6, "y":0},
+				{"label":"&", "x":7, "y":0},
+				{"label":"*", "x":8, "y":0},
+				{"label":"(", "x":9, "y":0},
+				{"label":")", "x":10, "y":0},
+				{"label":"_", "x":11, "y":0},
+				{"label":"+", "x":12, "y":0},
+				{"label":"Bksp", "x":13, "y":0},
+				{"x":14, "y":0},
+				{"label":"Num", "x":15.5, "y":0},
+				{"label":"/", "x":16.5, "y":0},
+				{"label":"*", "x":17.5, "y":0},
+				{"label":"-", "x":18.5, "y":0},
+				{"label":"Tab", "x":0, "y":1, "w":1.5},
+				{"label":"Q", "x":1.5, "y":1},
+				{"label":"W", "x":2.5, "y":1},
+				{"label":"E", "x":3.5, "y":1},
+				{"label":"R", "x":4.5, "y":1},
+				{"label":"T", "x":5.5, "y":1},
+				{"label":"Y", "x":6.5, "y":1},
+				{"label":"U", "x":7.5, "y":1},
+				{"label":"I", "x":8.5, "y":1},
+				{"label":"O", "x":9.5, "y":1},
+				{"label":"P", "x":10.5, "y":1},
+				{"label":"{", "x":11.5, "y":1},
+				{"label":"}", "x":12.5, "y":1},
+				{"label":"|", "x":13.5, "y":1, "w":1.5},
+				{"label":"7", "x":15.5, "y":1},
+				{"label":"8", "x":16.5, "y":1},
+				{"label":"9", "x":17.5, "y":1},
+				{"x":18.5, "y":1},
+				{"label":"Caps Lock", "x":0, "y":2, "w":1.75},
+				{"label":"A", "x":1.75, "y":2},
+				{"label":"S", "x":2.75, "y":2},
+				{"label":"D", "x":3.75, "y":2},
+				{"label":"F", "x":4.75, "y":2},
+				{"label":"G", "x":5.75, "y":2},
+				{"label":"H", "x":6.75, "y":2},
+				{"label":"J", "x":7.75, "y":2},
+				{"label":"K", "x":8.75, "y":2},
+				{"label":"L", "x":9.75, "y":2},
+				{"label":":", "x":10.75, "y":2},
+				{"label":"\"", "x":11.75, "y":2},
+				{"label":"Enter", "x":12.75, "y":2, "w":2.25},
```

Your layout here doesn't include the key for `KC_NUHS`, but your layout macro in `m0lly.h` does, as does the default keymap (assigned to `KC_NO`/`XXXXXXX`).

```suggestion
				{"label":"ISO ~", "x":12.75, "y":2},
				{"label":"Enter", "x":13.75, "y":2, "w":1.25},
```